# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/bumprace/hotfix-c23.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Fix the build with C23: StartText() takes no arguments.

diff -Nurp bumprace-1.5.8.orig/src/bumprace.c bumprace-1.5.8/src/bumprace.c
--- bumprace-1.5.8.orig/src/bumprace.c	2019-01-09 17:36:11.000000000 +0100
+++ bumprace-1.5.8/src/bumprace.c	2026-09-27 18:20:24.214412692 +0200
@@ -1178,7 +1178,7 @@ int main(int argc, char *argv[])
   sprintf(text,"%s/font.scl",DATAPATH);
   Font=LoadImage("font.png",3);
   InitFont(Font);
-  if (final) StartText(i);
+  if (final) StartText();
 //load data
   printf("** Loading Data **\n");
 #ifdef SOUND
